Interview Card/Page

JackMcCormick opened the pull request:
Cards are now creating dynamically.
Not implemented:  Comments, Routing, Currently pulls from seed data

JackMcCormick committed:
Interview Card/Page

Cards are now creating dynamically.
Not implemented:  Comments, Routing, Currently pulls from seed data

JackMcCormick committed:
Merge branch '1911-trevin-front-end-dev' into 1911-Trevin-Interview-Card

jaychaudhary1 reviewed (changes requested):
please fix httpservice instead of interviewservice

JackMcCormick committed:
Fixing Interview Card Info

Now pulls from database call instead of dummy data. I left the dummy data call in commented

JackMcCormick committed:
Merge branch '1911-Trevin-Interview-Card' of https://github.com/revaturelabs/interview-tracker-frontend into 1911-Trevin-Interview-Card

JackMcCormick committed:
Update interviews.component.ts

jaychaudhary1 reviewed (approved):
noice very noice
